cmd/zed/zed.d/zed.rc

```diff
@@ -147,3 +147,11 @@ ZED_SYSLOG_SUBCLASS_EXCLUDE="history_event"
 # help silence misbehaving drives.  This assumes your drive enclosure fully
 # supports slot power control via sysfs.
 #ZED_POWER_OFF_ENCLOUSRE_SLOT_ON_FAULT=1
+
+# During auto-replace of vdev device, the device is labeled
+# asynchronously on Linux. Use ZED_DISK_LABEL_WAIT_TIME to
+# change how long to wait for the partitioned device symlinks
+# to show up before giving up. The time is in milliseconds and
+# defaults to 3000 if not specified here.
+#
+#ZED_DISK_LABEL_WAIT_TIME=3000
```

Comment:
If the default is now 30,000 ms right?

Reply:
No, the default for ZED is still 3000 ms.  And the default for `zpool create`, i.e., `DISK_LABEL_WAIT`, remains 30,000 ms. This PR currently is only making it tunable.

In the ZED case, it is waiting after consuming a disk add event for the partition (after the partition was made and `sdh1` exists, but other links like under `/dev/disk/by-id/` might be inflight) , whereas in the `zpool create` case it is waiting just after issuing the partition request (before partitioning may have even started) so it makes sense that it is  a bit longer.

But you bring up a good point, the wait time is the maximum time it will wait so why not make the default longer for an auto-replace?

Reply:
It does seem like we are here fixing this because the wait is too short, so yeah, changing the default while here seems like a good idea